source/_integrations/juicenet.markdown

```diff
@@ -22,8 +22,11 @@ ha_integration_type: integration
 
 The `juicenet` platform pulls data from a [JuiceNet](https://evcharging.enelx.com/products/juicebox) charging station equipped with a Wi-Fi connection. It will access and make available all of the devices attached to your account. It also exposes a switch allowing you to charge your car now instead of waiting for the pre-set schedule.
 
+> **⚠️ Warning:** This integration can no longer be set up as the required API token cannot be obtained. However, if you have it working, it may remain functional.
```

Comment:
```suggestion
{% important %}
This integration can no longer be set up as the required API token cannot be obtained. However, if you have it working, it may remain functional.
{% endimportant %}
```